docs/user-guide/deployments.md

```diff
@@ -225,7 +229,11 @@ before changing course.
 Sometimes you may want to rollback a Deployment; for example, when the Deployment is not stable, such as crash looping.
 By default, all of the Deployment's rollout history is kept in the system so that you can rollback anytime you want 
 (you can change that by specifying [revision history limit](/docs/user-guide/deployments/#revision-history-limit)). 
-You should note that only the update to the Deployment's "pod template" is seen as a different revision of rollout.
+
+**Note:** a Deployment's revision is created when a Deployment's rollout is triggered. This means that the new revision is created 
+if and only if the Deployment's pod template (i.e. `.spec.template`) is changed, e.g. updating labels or container images of the template labels. 
+Other updates, such as scaling the Deployment, will not create a Deployment revision. This implies that when you rollback to an earlier revision, 
+only the Deployment's pod template part will be rolled back. 
```

Comment:
A little explanation of why this is (to facilitate simultaneous manual or auto-scaling) would be helpful.